src/main/java/edu/harvard/iq/dataverse/DatasetFieldServiceBean.java

```diff
@@ -321,14 +322,15 @@ public Map<Long, JsonObject> getCVocConf(boolean byTermUriField){
                               + jo.getString("term-uri-field"));
                         }
                     }
-                    if (jo.containsKey("child-fields")) {
-                        JsonArray childFields = jo.getJsonArray("child-fields");
-                        for (JsonString elm : childFields.getValuesAs(JsonString.class)) {
-                            dft = findByNameOpt(elm.getString());
-                            logger.info("Found: " + dft.getName());
+                    if (jo.containsKey("managed-fields")) {
```

Comment:
Just confirming - this was a bug? - the schema requires managed-fields these days

Reply:
Not a bug, I would say old code, `child-fields` does not exists anymore so I replaced it with `managed-fields`. I think this code is not useful but at least the key is not obsolete.

Reply:
Right - looks like it is just for logging purposes/to warn about incorrect field choices.